I can reproduce with ` nix-build --option sandbox true -A blastem`

```
...
true ./blastem
gcc -O2 -flto -I/nix/store/cbcbqfpjkxpidsqv3r7w7slgik7fwx93-SDL2-2.0.8-dev/include/SDL2 -I/nix/store/4x4l8q7qisjg1gabvpq99cypr646sfgb-mesa-noglu-18.1.5-dev/\
include -I/nix/store/2yabd23z60jjgw1bvbnmmjlcp58xx2nw-glew-2.1.0-dev/include -I/nix/store/bil4p8ry2haarf1yhcz5vy9m0jwxnzcs-glu-9.0.0-dev/include -std=gnu99 \
-Wreturn-type -Werror=return-type -Werror=implicit-function-declaration -Wno-unused-value  -DX86_64 -m64  -O2 -flto -lm -L/nix/store/b028c6ax8622375lp2gjc3q\
6xznpkdbr-SDL2-2.0.8/lib -L/nix/store/3v1rcfkyzrf07vhij95cy9j72qx63y5j-libglvnd-1.0.0/lib -L/nix/store/v50b6limw49rcg686akm4aidxrawmymf-glew-2.1.0/lib -L/ni\
x/store/h2ipbyqfg712x16qsagpanybpxs3249r-glu-9.0.0/lib -Wl,-rpath,/nix/store/b028c6ax8622375lp2gjc3q6xznpkdbr-SDL2-2.0.8/lib -Wl,--enable-new-dtags -lSDL2 -\
lGLEW -lGLU -lGL -m64  termhelper.c   -o termhelper
/bin/sh: /bin/echo: not found
xcf2png font_interlace_variable.xcf > font_interlace_variable.png
./img2tiles.py -s font_interlace_variable.spec font_interlace_variable.tiles
make: ./img2tiles.py: Command not found
make: *** [Makefile:236: font_interlace_variable.tiles] Error 127
builder for '/nix/store/h8cnfldwg9v8g41l565biy08ln95damh-blastem-0.5.1.drv' failed with exit code 2
error: build of '/nix/store/h8cnfldwg9v8g41l565biy08ln95damh-blastem-0.5.1.drv' failed
```

--- a/pkgs/misc/emulators/blastem/default.nix
+++ b/pkgs/misc/emulators/blastem/default.nix
@@ -1,0 +1,48 @@
+{ stdenv, fetchurl, fetchFromGitHub, pkgconfig, SDL2, glew, xcftools, python, pillow, makeWrapper }:
+
+let
+  vasm =
+    stdenv.mkDerivation rec {
+      name = "vasm-${version}";
+      version = "1.8c";
+      src = fetchFromGitHub {
+        owner = "mbitsnbites";
+        repo = "vasm";
+        rev = "244f8bbbdf64ae603f9f6c09a3067943837459ec";
+        sha256 = "0x4y5q7ygxfjfy2wxijkps9khsjjfb169sbda410vaw0m88wqj5p";
+      };
+      makeFlags = "CPU=m68k SYNTAX=mot";
+      installPhase = ''
+        mkdir -p $out/bin
+        cp vasmm68k_mot $out/bin
+      '';
+    };
+in
+stdenv.mkDerivation rec {
+  name = "blastem-${version}";
+  version = "0.5.1";
+  src = fetchurl {
+    url = "https://www.retrodev.com/repos/blastem/archive/3d48cb0c28be.tar.gz";
+    sha256 = "07wzbmzp0y8mh59jxg81q17gqagz3psxigxh8dmzsipgg68y6a8r";
+  };
+  buildInputs = [ pkgconfig SDL2 glew xcftools python pillow vasm makeWrapper ];
+  preBuild = ''
+    patchShebangs img2tiles.py
+  '';
+  postBuild = ''
+    make menu.bin
+  '';
+  installPhase = ''
+    mkdir -p $out/bin $out/share/blastem
+    cp -r {blastem,menu.bin,default.cfg,rom.db,shaders} $out/share/blastem/
+    makeWrapper $out/share/blastem/blastem $out/bin/blastem
+  '';
+
+  meta = {
+    homepage = https://www.retrodev.com/blastem/;
+    description = "The fast and accurate Genesis emulator";
+    maintainers = with stdenv.lib.maintainers; [ puffnfresh ];
+    license = stdenv.lib.licenses.gpl3;
+    platforms = stdenv.lib.platforms.linux;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -1659,6 +1659,10 @@ with pkgs;
 
   biber = callPackage ../tools/typesetting/biber { };
 
+  blastem = callPackage ../misc/emulators/blastem {
+    inherit (python27Packages) pillow;
+  };
+
   blueman = callPackage ../tools/bluetooth/blueman {
     withPulseAudio = config.pulseaudio or true;
   };

